utils.js

```diff
@@ -183,25 +183,60 @@ CTLEventUtils.readURLParams = function(eventsList, queryString, index) {
             case 'q':
                 filteredEvents = CTLEventUtils.searchEvents(
                     filteredEvents, index, splitParam[1]);
+                CTLEventUtils.populateQ(splitParam[1]);
```

Comment:
`readURLParams` does two things: it reads in the params and then filters them.  What do you think about breaking that apart, such that:
- readURLParams() would return an array of key-value pairs from the URL
- filterOnURLParams() takes in the array of key-value pairs and an array of events, returns an array of filtered events
- populateURLParams() takes in the array of key-value pairs and populates the fields